demisto_sdk/commands/format/format_module.py

```diff
@@ -203,14 +213,16 @@ def format_manager(
             file_path = str(Path(file))
             file_type = find_type(file_path, clear_cache=clear_cache)
 
-            current_excluded_files = excluded_files[:]
-            dirname = os.path.dirname(file_path)
-            if dirname.endswith("CommonServerPython"):
-                current_excluded_files.remove("CommonServerPython.py")
-            if os.path.basename(file_path) in current_excluded_files:
-                continue
-            if any(test_dir in str(dirname) for test_dir in TESTS_AND_DOC_DIRECTORIES):
-                continue
+            # Check if this is an unskippable file
+            if not any(
+                [
+                    file_path.endswith(unskippable_file)
+                    for unskippable_file in UNSKIP_FORMATTING_FILES
+                ]
+            ):
+                # If it is not an unskippable file, skip if needed
+                if os.path.basename(file_path) in SKIP_FORMATTING_FILES:
+                    continue
```

Comment:
Do we have a unit test that check it fo all skippable and unskippable files today?

Reply:
Unskippable: test_format_commonserver_skipped_files in format_integration_test.py
Skippable: test_format_integration_skipped_files in format_integration_test.py